src/commands/synthetics/reporters/default.ts

```diff
@@ -125,7 +134,7 @@ const renderResultOutcome = (result: Result, test: Test, icon: string, color: ch
   }
 
   if (test.type === 'browser') {
-    if (!hasResultPassed(result) && result.stepDetails) {
+    if (!hasResultPassed(result, allowNetworkIssue) && result.stepDetails) {
```

Comment:
According to `hasResultPassed` definition lower (https://github.com/DataDog/datadog-ci/pull/250/files#diff-741a985551714cec0c6f7a6423d704525619e8f57db6c4f7c685d95d88a8d0cfR140) it should be taking `allowOnUnexpectedResults` instead.
Here it looks like it should not depend on `allowNetworkIssue`